### PR TITLE
Big data and data science updates

### PR DESCRIPTION
**Pull request by rvs**

### CODE REVIEW

File: _posts/2015-01-01-intro.md [hunk 1]
```diff
@@ -31,12 +31,11 @@ before](https://archive.fosdem.org/2014/schedule/track/hpc_and_computational_sci
 but this is the first time that HPC and Big Data open-source enthusiasts are
 joining forces to bring the two communities together with this devroom.
 
-- HPC and Big Data are two important approaches to scientific computing. HPC largely focused on the CPU bottlenecks while Big Data largely focuses on the IO bottlenecks. We welcome talks about Free and Open Source solutions to the challenges of large scale scientific computing.
+- HPC and Big Data are two important approaches to scientific computing. HPC typically deals with smaller, higly structured data sets and huge amounts of computation while Big Data, not surprisingly, deals with gigantic, unstructured data sets and focuses on the IO bottlenecks. With Big Data trend unlocking access to an unprecedented amount of data, Data Science has emerged to tackle the problem of creating processes and approaches to extracting knowledge or insights from these data sets. Machine learning and predictive analytics are now the predominant type of computation performed on large clusters. We welcome talks about Free and Open Source solutions to the challenges presented by of large scale computing, data management and data analysis.
```
Comment (vasia) [inline _posts/2015-01-01-intro.md:35]: "presented by of large scale" -> remove "of"

**Comment by vasia** [comment]
+1 from me, thanks @rvs!

**Comment by ehiggs** [comment]
:+1:

**Comment by boegel** [comment]
@rvs: I'm merging this, will issue a follow up PR with some minor changes (I noticed a typo, for example)

**Pull request merged**